### PR TITLE
feat(ReportsTable): new compliance filters

### PR DESCRIPTION
**Pull request by Fewwy**
Two new Compliance filters for the Reports table

image

image

### CODE REVIEW

**Comment by Victoremepunto** [comment]
How did the commitlint task passed for this commit message ? :thinking: - there's no mention to the JIRA task

**Comment by skateman** [comment]
> How did the commitlint task passed for this commit message ? thinking - there's no mention to the JIRA task

we don't require a JIRA ticket in the commit message, but it would be nice to have it here :see_no_evil:

**Comment by xmicha82** [comment]
hey, is this correct behaviour?
image

**Comment by LightOfHeaven1994** [comment]
LGTM, thanks @Fewwy !

**Review by LightOfHeaven1994 — APPROVED** [review]

**Review by LightOfHeaven1994 — CHANGES_REQUESTED** [review]
Screenshot from 2022-08-04 11-31-51
I was able to reproduce bug that Martin attached

**Comment by Fewwy** [comment]
@LightOfHeaven1994 Ok I'm looking in to the why the bug is happening

**Comment by skateman** [comment]
It might be a backend issue, I'm investigating.

**Comment by Fewwy** [comment]
@skateman Any updates?

**Comment by skateman** [comment]
I'm deep in debugging a memory leak and our SLO, so this has a lower prio atm.

File: src/constants.js [hunk 1]
```diff
@@ -170,10 +170,12 @@ export const COMPLIANT_SYSTEMS_FILTER_CONFIGURATION = [
   {
     type: conditionalFilterType.checkbox,
     label: 'Compliance',
-    filterString: (value) => `compliant = ${value}`,
+    filterString: (value) => `${value}`,
     items: [
-      { label: 'Compliant', value: 'true' },
-      { label: 'Non-compliant', value: 'false' },
+      { label: 'Compliant', value: 'compliant = true' },
+      { label: 'Non-compliant', value: 'compliant = false' },
```
Comment (skateman) [inline src/constants.js:176]: I think what @LightOfHeaven1994 is describing can be fixed by these changes:
```suggestion
      { label: 'Compliant', value: 'compliant = true AND supported_ssg = true' },
      { label: 'Non-compliant', value: 'compliant = false AND supported_ssg = true' },
```

**Review by skateman — COMMENTED** [review]
Not sure if UX would approve it, but I think we should restrict these filters a little bit. Ideally by allowing only one to be selected from these. Is that something doable? We're very limited here on the backend :disappointed:

File: src/constants.js (hunk 1 above)
Comment (LightOfHeaven1994) [reply src/constants.js:176]: @Fewwy can you try David's suggestion? also PR needs to be rebased as we use new tags gql query

Comment (skateman) [reply src/constants.js]: @LightOfHeaven1994 he's on PTO, I rebased and adjusted the PR, can you please take a look?

Comment (LightOfHeaven1994) [reply src/constants.js]: @skateman Works great now! thanks for the fix <3

**Review by LightOfHeaven1994 — APPROVED** [review]

**Pull request merged**